axelrod/tests/unit/test_fingerprint.py

```diff
@@ -1,7 +1,24 @@
+"""
+added or modified tests
+
```

Comment:
the lion's share of tests are simply in different places, and the ones not in TestAshlockFingerprint test their original data, but it's in the test.

anything not EXACTLY like it was is marked.